controllers/util/solr_util.go

```diff
@@ -607,12 +615,20 @@ func generateSolrSetupInitContainers(solrCloud *solr.SolrCloud, solrCloudStatus
 		}
 	}
 
+	volumePrepResources := corev1.ResourceList{
+		corev1.ResourceCPU:    *DefaultSolrVolumePrepInitContainerCPU,
```

Comment:
👏This will help to make sure if we configure our solr resources with limit=request as well,  the pod will get assigned with QoS class Guaranteed